src/taipy/gui/data/data_scope.py

```diff
@@ -47,14 +47,6 @@ def get_scope(self, client_id: t.Optional[str]) -> SimpleNamespace:
     def get_all_scopes(self) -> t.Dict[str, SimpleNamespace]:
         return self.__scopes
 
-    def broadcast_data(self, name, value):
-        if self.__single_client:
-            return
-        if not self.__client_id:
-            for _, v in self.__scopes.items():
```

Comment:
can we use values() here ?

Reply:
certainly yes. Didn't optimize yet. Thanks Fred!